pkgs/development/compilers/purescript/purescript/default.nix

```diff
@@ -62,8 +58,8 @@ in stdenv.mkDerivation rec {
     homepage = "https://www.purescript.org/";
     license = licenses.bsd3;
     maintainers = with maintainers; [ justinwoo mbbx6spp cdepillabout ];
```

Comment:
@a-m-joseph One thing that would be a MUST for me before we merge this is for you to add yourself as a maintainer, and do things like the following:

- update this `purescript` derivation in a timely manner whenever there is a new release upstream
- respond to issues / PRs here in Nixpkgs about PureScript (although honestly there are few issues or PRs)

I'd also like to hear @mbbx6spp's opinion about this PR.